rskj-core/src/main/java/co/rsk/net/SyncProcessor.java

```diff
@@ -289,8 +289,8 @@ public void startBlockForwardSyncing(Peer peer) {
     }
 
     @Override
-    public void startSnapSync() {
-        logger.info("Start Snap syncing");
+    public void startSnapSync(Peer peer) {
```

Comment:
question:

Does it mean that we will accept the sync only with a single peer for now right? 🤔 
I don't recall if it was possible to receive from multiple peers.

Reply:
yes, we start with one peer - we send `SnapStatusRequestMessage` to get initial params for syncing, like state root hash, etc.